### ISSUE TITLE
[Options tile] Initial design exploration a11y

### ISSUE THREAD

**Issue by RichKummer** [carbon-design-system/ibm-products#7606]
Discussion from 06/02/25 Carbon Accessibility guild. Came up in review of WC Options tile #7139 

Options tile currently has an unintuitive tab experience – the entire accordion is tabbable and clickable, but a toggle appears inside the accordion. This potentially creates accessibility issues with knowing what you are clicking or tabbing to. One option might be to shift to an icon button instead of an accordion thats entire area is clickable. 

@elycheea mentioned that when the the whole header was clickable and drops to icon only, it does feel like a regression in the implementation. We should explore other options as well.

[Current design docs](https://pages.github.ibm.com/carbon/ibm-products/components/options-tile/usage/#variants)

**David's example that uses an icon button**
<img width="893" alt="Image" src="https://github.com/user-attachments/assets/9d9cd2ef-fa65-49aa-a9c9-38aa85d5ccf7" />

## Tasks:
- [x] Explore the current tab ordering of Options tile against a new version that uses an icon button on the left instead of a tab stop that covers the entire accordion.
- [x] Explore additional options in Figma that we could try for the Options tile. For example, can the focus for the accordion cover the entire area _except_ for the toggle? What if the focus for the accordion stops before the toggle?
- [x] A good resource is the Figma kit design analytics where you can can see how users are using Options tile. It might be worth time boxing to see how product teams use Options tile.
- [x] Reach out to Michael Gower to review both options. 
- [x] Review options with design and development and collaborate on a path forward.

**Comment by davidmenendez** [comment]
@RichKummer any update here? Michael Gower left some additional notes recently in the [a11y-guild channel](https://ibm-studios.slack.com/archives/G01GCBCGTPV/p1748892658980539) and he was looking for some feedback.

**Comment by RichKummer** [comment]
Thanks for the heads up @davidmenendez ! Judith's taking a look at Michael Gower's feedback from that thread to incorporate into the design exploration
